scripts/base.js

```diff
@@ -107,4 +107,53 @@ const updateCoordinates = async (
   return Ok("File updated successfully!");
 };
 
-export { Ok, Err, determineArgs, updateCoordinates };
+const updateParkingLots = async (
+  cityId,
+  addCity,
+  originalFilePath,
+  updateFilePath
+) => {
+  let newData;
+  try {
+    const rawNewData = await fs.readFile(originalFilePath, "utf8");
+    newData = JSON.parse(rawNewData);
+  } catch (err) {
+    return Err(
+      `Issue reading the update file path parking-lots-update.geojson: ${err.message}`
+    );
+  }
+
+  if (!Array.isArray(newData.features) || newData.features.length !== 1) {
+    return Err(
+      "The script expects exactly one entry in `features` because you can only update one city at a time."
+    );
+  }
+
+  const newCoordinates = newData.features[0].geometry.coordinates;
+  const newGeometryType = newData.features[0].geometry.type;
+
+  if (addCity) {
```

Comment:
Better is to invert the conditional and early return, like:

```
if (!addCity) {
  return
}

const newFile = ...
```

See https://medium.com/swlh/return-early-pattern-3d18a41bba8 about this tip.